css/styles.css

```diff
@@ -0,0 +1,118 @@
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+.container {
+  width: 965px;
+  margin: auto;
+}
+header{
+  height: 200px;
+  background-color: black;
+  width: 100%;
+  color: white;
+  border-bottom: 5px solid yellow;
+}
+h1 {
+   padding: 40px;
+   font-size: 30px;
+   color: yellow;
+}
+
+nav {
+  float: right;
+}
+li {
+  display: inline;
+  padding: 10px;
+  font-family: 'Yanone Kaffeesatz', Helvetica, Arial;
+  font-size: 26px;
+}
+.subheader {
+  height: 100px;
+  background-color: #BBB8B8;
+}
+h5 {
+  text-align: center;
+  font-size: 36px;
+  padding-top: 30px;
+  font-style: italic;
+  font-weight: bolder;
+}
+.photobody {
+  height: 700px;
+  background: url(../images/coworking.jpg) no-repeat center center rgb(134,141,157);
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  background-size: cover;
+}
+h2 {
+  font-weight: bold;
+  font-size: 26px;
+  padding: 0 0 20px;
+}
+.textbox {
+  float: right;
+  width: 400px;
+  height: 500px;
+  background-color: rgba(255,255,255,0.7);
+  font-size: 22px;
+  line-height: 30px;
+  padding: 40px;
+}
+.textbox button {
```

Comment:
try using classes for general button styling
and for the yellow one, use specific selector to change color only